go/ct/gen/code_test.go

```diff
@@ -354,6 +354,142 @@ func TestCodeGenerator_ClonesCanBeUsedToResetGenerator(t *testing.T) {
 	}
 }
 
+func TestCodeGenerator_CodeIsLargeEnoughForAllConditionOps(t *testing.T) {
+
+	type constOp struct {
+		p  int
+		op OpCode
+	}
+
+	type varOp struct {
+		v  string
+		op OpCode
+	}
+
+	tests := map[string]struct {
+		offset       int
```

Comment:
The name `offset` does not convey any information about the purpose of this field.

Looking at the implementation below, it looks like it is the forced code length. So why not name it `codeLength` or `codeSize`?